deltachat-ffi/deltachat.h

```diff
@@ -1447,6 +1447,55 @@ int             dc_set_chat_mute_duration             (dc_context_t* context, ui
 char*           dc_get_msg_info              (dc_context_t* context, uint32_t msg_id);
 
 
+/**
+ * Get uncut text of message, if available.
```

Comment:
> For simplicity, the function _always_ returns HTML-code,

Is `text` the right word here?

Reply:
you're right, "text" is too specific, also images, formatting might be cut.  i'll change that to "Get uncut message"